components/atom/button/src/config.js

```diff
@@ -18,6 +18,11 @@ export const DESIGNS = {
   LINK: 'link'
 }
 
+export const ALIGN_TEXT = {
```

Comment:
why the right alignment is not offered?

Reply:
better call it 'ALIGNEMENT', also for the prop but lower case.

Reply:
The comment about the RIGHT alignment is valid for consistency, although we don't have right to left languages in MA, so it's hard to imagine a use case fo it